Makefile

```diff
@@ -55,6 +57,11 @@ LLVM_CXX_FLAGS += -DLLVM_VERSION=$(LLVM_VERSION_TIMES_10)
 WITH_NATIVE_CLIENT ?= $(findstring nacltransforms, $(LLVM_COMPONENTS))
 WITH_X86 ?= $(findstring x86, $(LLVM_COMPONENTS))
 WITH_ARM ?= $(findstring arm, $(LLVM_COMPONENTS))
+ifeq ($(LLVM_VERSION_TIMES_10),39)
```

Comment:
I think we probably should hold off on the merge for it, it will break travis if it is merged as-is, even for non-Hexagon targets (on LLVM 3.8). That said, I doubt this is going to be too hard to fix, just annoying. I'll take another look at it later today.